data/petsData.js

```diff
@@ -1,15 +1,333 @@
+const { ObjectId } = require('mongodb');
```

Comment:
remove this { ObjectId }

Reply:
We need this for ObjectId conversions when validating the existence of owners in the user collection